src/base/unicodestrings.h

```diff
@@ -35,6 +35,7 @@
 // we put all problematic UTF-8 chars/strings in this file.
 // See issue #3059 for more details (https://github.com/qbittorrent/qBittorrent/issues/3059).
 const char C_INFINITY[] = "∞";
+const char C_NON_BREAKING_SPACE[] = " ";
```

Comment:
I don't recall a thing... 

But I do recall I tested `C_NON_BREAKING_SPACE` by putting it into a QLabel and found out Qt doesn't respect it, maybe my memory fails me, better test it yourself.